syntax = "proto3";

package common.v1;

message S3File {
  string key = 1 ;
}
